src/com/lftechnology/java/training/dipak/employeemanagement/dao/DbConnection.java

```diff
@@ -0,0 +1,46 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.dao;
+
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p>This class has a static method for establishing database connection.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class DbConnection {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)